mrmustard/physics/bargmann.py

```diff
@@ -12,8 +12,131 @@
+
+.. math:: 
+    |z\rangle_b = e^\frac{|z|^2}{2}|z\rangle_c = \sum_n \frac{z^n}{\sqrt{n!}}|n\rangle_f.
+
+Any Gaussian objects :math:`O` can be written in the Bargmann basis as a Gaussian exponential function, parametrized by a matrix :math:`A`, a vector :math:`b` and a scalar :math:`c`, which is called ``triples`` through all the documentations in MM:
+
+.. math::
+    \langle\vec{\alpha}|O\rangle = c \exp\left( \frac12 \vec{\alpha}^T A \vec{\alpha} + \vec{\alpha}^T b \right).
+
+The objects in Bargmann representation uses the :class:`~mrmustard.physics.ansatze.PolyExpAnsatz` and the information is stored in the triple (A,b,c).
+
+The expression :math:`\langle\vec{\alpha}|O\rangle` is vectorized the variables vector :math:`\vec{\alpha}`, which is different for different quantum objects. 
+As for a ``n``-mode pure Gaussian state :math:`\langle\vec{\alpha}|\psi\rangle`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_{n-1})`.
+
+    .. code-block::
+
+        ╔═══════╗
+        ║       ║─────▶ alpha^*_0
+        ║ |psi> ║─────▶ alpha^*_1
+        ║       ║...
+        ║       ║─────▶ alpha^*_(n-1)   
+        ╚═══════╝    
+
+All the wires in ths mixed Gaussian state :math:`\langle\vec{\alpha}|\rho|\vec{\beta}\rangle`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_{n-1}, \beta_0, \beta_1,..., \beta_{n-1})`.
+
+    .. code-block::
+
+        ╔═══════╗
+        ║       ║─────▶ alpha^*_0
+        ║       ║─────▶ alpha^*_1
+        ║       ║─────▶ ...
+        ║       ║─────▶ alpha^*_(n-1)  
+        ║  rho  ║─────▶ beta_0_1
+        ║       ║─────▶ beta_1
+        ║       ║─────▶ ...
+        ║       ║─────▶ beta_(n-1)    
+        ╚═══════╝ 
+
+The wires in the diagram below correspond to the `out_bra` wires (:math:`\alpha^*`) and the `out_ket` wires (:math:`\beta`) in :class:`~mrmustard.lab_dev.wires.Wires`.
+
+As for a ``n``-mode Gaussian unitary :math:`\langle\vec{\alpha}|U|\vec{\beta}\rangle`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_{n-1})`.
+
+    .. code-block::
+
+                        ╔═══════╗
+        beta_0    ─────▶║       ║─────▶ alpha^*_0
+        beta_1    ─────▶║   U   ║─────▶ alpha^*_1
+                     ...║       ║...
+        beta_(n-1)─────▶║       ║─────▶ alpha^*_(n-1)       
+                        ╚═══════╝    
+
+The wires in the diagram below correspond to the `out_ket` wires (:math:`\alpha^*`) and the `in_ket` wires (:math:`\beta`) in :class:`~mrmustard.lab_dev.wires.Wires`.
+                    
+As for a ``n``-mode Gaussian Channel :math:`\langle \vec{\alpha}|\Psi(|\vec{\gamma}\rangle\langle\vec{\delta}|)|\vec{\beta}`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_{n-1}, \beta_0, \beta_1,..., \beta_{n-1}, \delta^*_0, \delta^*_1, ..., \delta^*_{n-1}, \gamma_0, \gamma_1,..., \gamma_{n-1})`.
```

Comment:
for channels the default order is out,in,out,in so alpha, delta, beta, gamma in your notation